packages/bot-skeleton/src/services/tradeEngine/Interface/BotInterface.js

```diff
@@ -9,8 +9,6 @@ const getBotInterface = tradeEngine => {
         start: (...args) => tradeEngine.start(...args),
         stop: (...args) => tradeEngine.stop(...args),
         purchase: contract_type => tradeEngine.purchase(contract_type),
-        getAskPrice: contract_type => Number(getProposal(contract_type, tradeEngine).ask_price),
-        getPayout: contract_type => Number(getProposal(contract_type, tradeEngine).payout),
```

Comment:
Let's not remove these functions and instead point them to an empty function or constant number for now.

Reply:
done✅